### ISSUE TITLE
Dropwizard Metrics 3.x and 4.1.x are EOL

### ISSUE THREAD

**Issue by MikaelSmith** [joshelser/dropwizard-hadoop-metrics2#7]
Metrics 4.1.x and earlier are [EOL](https://github.com/dropwizard/metrics/discussions/3029). We should update this project to work with Metrics 4.2.

@joshelser are you interested in contributions?

**Comment by joshelser** [comment]
Of course! I don't have much time for this anymore, but happy to help out as I can.
